### ISSUE TITLE
flatpak-pip-generator: subprocess "flatpak run" fails without platform version

### ISSUE THREAD

**Issue by tedb** [flatpak/flatpak-builder-tools#142]
Hello, I am new to Flatpak, and looking forward to using `flatpak-pip-generator` to hopefully package up OctoPrint.  I encountered a confusing error running the script, but it does appear to have a workaround.

In short, it looks like `flatpak run org.freedesktop.Platform` defaults to a version called `master`, but the version I have from Flathub is `19.08`.

Here is a complete reproduction, on Debian Buster:

```shell
$ flatpak --version
Flatpak 1.2.5

$ wget -q https://github.com/flatpak/flatpak-builder-tools/raw/master/pip/flatpak-pip-generator

$ python3 flatpak-pip-generator
Traceback (most recent call last):
  File "flatpak-pip-generator", line 138, in <module>
    result = subprocess.run(command, check=True, capture_output=True)
  File "/usr/lib/python3.7/subprocess.py", line 487, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['flatpak', 'run', '--command=python3', 'org.freedesktop.Platform', '--version']' returned non-zero exit status 1.

$ flatpak run --command=python3 org.freedesktop.Platform --version
error: app/org.freedesktop.Platform/x86_64/master not installed

# Specifying the version as 19.08 works as intended (shows Python version)
$ flatpak run --command=python3 org.freedesktop.Platform//19.08 --version
Python 3.7.4

# Add 19.08 to the platform name for "flatpak run"
$ sed -i "s/'org.freedesktop.Platform'/'org.freedesktop.Platform\/\/19.08'/" flatpak-pip-generator

# Scripts executes now, as intended
$ python3 flatpak-pip-generator
Please specifiy either packages or requirements file argument
```

Would it make sense to update the script to use a specific version of the platform instead of `master`, or is it recommended to run the `master` version instead of a version like `19.08`?  I couldn't figure out how to get a "master" version, having tried the following to no avail:

```
$ flatpak install flathub org.freedesktop.Platform//master
Looking for matches…
error: Nothing matches org.freedesktop.Platform in remote flathub
```

**Comment by TingPing** [comment]
@A6GibKm

**Comment by A6GibKm** [comment]
Fixed by #143, the idea was to check what python version was inside of the sandbox and compare it with the local version, ideally this should never break the script.
